src/microbe_stage/organelle_unlocks/OrganelleUnlockHelpers.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnlockConstraints;
+
+public static class OrganelleUnlockHelpers
+{
+    private static IEnumerable<IUnlockCondition>? allUnlockConditions;
```

Comment:
This should not be global state. This should be instead stored in the GameWorld. And I guess that would basically remove the need for this class entirely so I'm in favour of rolling all of the code in this file into the GameWorld class directly (or another place if you can think of some place more sensible).

Reply:
I'd probably say UnlockProgress would be more sensible